pkg/server/datastore/sqlstore/migration.go

```diff
@@ -353,6 +355,8 @@ func migrateVersion(tx *gorm.DB, currVersion int, log logrus.FieldLogger) (versi
 	switch currVersion {
 	case 17:
```

Comment:
Not required, but opportunistically, this test case, and the `migrateToV18` function can be removed since anyone upgrading to v1.3.x will have to already have upgraded through v.1.2.x, which will have already performed the migration.